test/test_sparse.py

```diff
@@ -1694,6 +1694,184 @@ def test_log1p(self):
             device=self.device)
         self._test_log1p_tensor(input, torch.zeros([5, 6, 0]))
 
+    def _test_abs_absolute(self, sparse_tensor, dense_tensor):
+        expected_output = dense_tensor.abs()
+        self.assertEqual(expected_output, abs(sparse_tensor).to_dense())
+        self.assertEqual(expected_output, torch.abs(sparse_tensor).to_dense())
+        self.assertEqual(expected_output, sparse_tensor.abs().to_dense())
+        self.assertEqual(expected_output, sparse_tensor.coalesce().abs_().to_dense())
+
+        self.assertEqual(expected_output, torch.absolute(sparse_tensor).to_dense())
+        self.assertEqual(expected_output, sparse_tensor.absolute().to_dense())
+        self.assertEqual(expected_output, sparse_tensor.coalesce().absolute_().to_dense())
+
```

Comment:
`abs` has `out` parameter. Could you test the usage of the `out` parameter as well?